pages/desktop/base.py

```diff
@@ -130,6 +130,9 @@ class HeaderRegion(Page):
         _account_dropdown_locator = (By.CSS_SELECTOR, "#aux-nav .account ul")
         _logout_locator = (By.CSS_SELECTOR, "li.nomenu.logout > a")
 
+        _tools_dropdown_locator = (By.CLASS_NAME, "tools")
+        _submit_a_new_addon_locator = (By.LINK_TEXT, "Submit a New Add-on")
```

Comment:
I'm okay with this for now, but could you raise an issue against addons-server to give this element a class or id that we can use for testing?